### PR TITLE
Fix optimism dupe assets

### PR DESCRIPTION
**Pull request by brunobar79**
Fixes RNBW-4082
Fixed RNBW-4285
Figma link (if any):

## What changed (plus any additional context for devs)
We don't need this anymore and it's causing dupe assets in some scenarios

## Screen recordings / screenshots

BEFORE:
IMG_4735

AFTER:
Screen Shot 2022-09-14 at 6 43 02 PM

## What to test

Import the swap aggregator wallet as watch only. Optimism assets shouldn't be duplicated.

## Final checklist

- [x] Assigned individual reviewers?
- [x] Added labels? (`team1/team2`, `critical path`, `release`, `dev QA`)
- [x] Did you test both iOS and Android?
- [x] If your changes are visual, did you check both the light and dark themes?
- [ ] Added e2e tests? If not, please specify why
- [ ] If you added new critical path files, did you update the CODEOWNERS file?
- [ ] If no `dev QA` label, did you add the PR to the QA Queue?

### CODE REVIEW

**Comment by pugson** [comment]
e2e failing

File: src/references/chain-assets.json [hunk 1]
```diff
@@ -16,368 +16,5 @@
       },
       "quantity": 0
     }
-  ],
-  "optimism": [
```
Comment (derHowie) [inline src/references/chain-assets.json:20]: Looks like basing `optimismExplorerInit` around `chain-assets.json` no longer makes sense. That should be addressed at some point, but, imo, you should leave this as `"optimism": []` to get the fix out and get passed the linter.

Comment (brunobar79) [reply src/references/chain-assets.json:20]: good call!

File: src/references/chain-assets.json [hunk 2]
```diff
@@ -16,368 +16,5 @@
...
-          "changed_at": 1582562452,
-          "relative_change_24h": 0.8470466197462612,
-          "value": 1
-        },
-        "symbol": "ENS",
-        "type": "optimism"
-      },
-      "quantity": 0
-    }
   ]
```
Comment (brunobar79) [inline src/references/chain-assets.json:19]:
```suggestion
  ],
  "optimism": []
```

**Review by derHowie — APPROVED** [review]

**Review by estebanmino — APPROVED** [review]
🔥

**Pull request merged**